core/vpgl/algo/vpgl_camera_compute.cxx

```diff
@@ -133,6 +136,31 @@ vpgl_affine_camera_compute::compute(
 }
 
 
+bool vpgl_affine_camera_compute::compute(vpgl_local_rational_camera<double> const& local_rcam, vgl_box_3d<double> region,
```

Comment:
This looks like a duplicate of vpgl_affine_camera_convert::convert()

Reply:
fair enough - removed the vpgl_affine_camera_compute